ensime_shared/client.py

```diff
@@ -421,6 +429,15 @@ def doc_uri(self, args, range=None):
         self.log.debug('doc_uri: in')
         self.send_at_position("DocUri", False, "point")
 
+    def usages(self, args, range=None):
```

Comment:
Oh yeah, I think it was my complaint about this that prompted creation of #244. Don't recall if there's a blocker to stop proliferating the pattern, if not changing all existing ones yet.